libs/rpc/protocols/object-protocol.js

```diff
@@ -0,0 +1,125 @@
+(function (window) {
+
```

Comment:
I do not see benefit of this protocol, since it is just repacking req/res to objects and vice versa. It does not do proper parsing / serialization, since it does not convert anything to string. If this design decision is taken in order to support some further development, then I think that _parse / _serialize methods should be empty and just pass around the same request / response. For example:

function _parse(request) { return request; }